lisp/pdf-view.el

```diff
@@ -405,6 +412,16 @@ PNG images in Emacs buffers."
   ;; Setup initial page and start display
   (pdf-view-goto-page (or (pdf-view-current-page) 1)))
 
+(when (fboundp 'advice-add)
+  (defun cua-copy-region--pdf-view-advice (&rest _)
+    "If current buffer is in `pdf-view' mode, call
+`pdf-view-kill-ring-save'."
+    (when (eq major-mode 'pdf-view-mode)
+      (pdf-view-kill-ring-save)))
+  (advice-add 'cua-copy-region
+	      :before-until
+	      #'cua-copy-region--pdf-view-advice))
+
```

Comment:
I think `pdf-view-kill-ring-save` might return `nil`, which means the function returns nil even if `pdf-view-mode` is true, which is not what you want I think.

Reply:
I just tried and `pdf-view-kill-ring-save` returns the copied string.